xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp

```diff
@@ -713,6 +741,8 @@ void CLinuxRendererGL::UpdateVideoFilter()
   bool pixelRatioChanged    = (CDisplaySettings::GetInstance().GetPixelRatio() > 1.001f || CDisplaySettings::GetInstance().GetPixelRatio() < 0.999f) !=
                               (m_pixelRatio > 1.001f || m_pixelRatio < 0.999f);
   bool nonLinStretchChanged = false;
+  bool cmsChanged           = (m_cmsOn != m_ColorManager->IsEnabled())
```

Comment:
My thinking here is that we need to check if cms has been toggled on/off or its configuration has changed. If cms is not on, CheckConfiguration is not called.

The second check misses the case where cms was just switched on and skips the call to CheckConfiguration, but that's ok since it will be checked later to see if a new CLUT texture needs to be loaded.

I did it this way to avoid reloading the CLUT texture when cms is toggled on/off repeatedly.